include/qinterface.hpp

```diff
@@ -315,6 +316,12 @@ class QInterface {
      *
      * If float rounding from the application of the matrix might change the state vector norm, "doCalcNorm" should be
      * set to true.
+     *
+     * \warning Physically, besides measurement, all quantum gates are unitary. This means there is always exactly a
+100% chance of finding a qubit in ANY STATE. A nonunitary gate besides measurement is nonphysical, at least because it
```

Comment:
Style adjustment needed.

Reply:
`make format` styles it this way, because of the `\warning`. That is how I want the documentation, though.